package.json

```diff
@@ -78,6 +78,7 @@
     "express": "^4.16.4",
     "file-loader": "^4.2.0",
     "file-saver": "^2.0.2",
+    "hookrouter": "^1.2.3",
```

Comment:
I was getting an error because the state object was not being accepted when clicking on the link generated by Link, but I went ahead and reverted it because the subsequent changes I made based on your above suggestions made it work using Link and now hookrouter is not necessary.

Reply:
Super. The fewer deps the better.